infrastructure/smart-contracts/test/NodeLicenseTinyKeys.mjs

```diff
@@ -98,7 +98,7 @@ export function NodeLicenseTinyKeysTest(deployInfrastructure, poolConfigurations
             await xai.connect(addr1).approve(await nodeLicense.getAddress(), priceInXai);
 
             // Mint an NFT
-            await nodeLicense.connect(addr1).mintWithXai(1, "", false, priceInEther);
+            await nodeLicense.connect(addr1).mintWithXai(1, "", false, priceInXai);
```

Comment:
These test cases were failing due to this bug with the wrong value.

Reply:
They were never passing ?

Reply:
I'm going from memory, but I think there was originally a bug in the contract around an operator being reversed in this logic. That was caught and fixed when the contract was deployed on Sepolia for testing, but seems the test cases were never updated.

I think I remember when we found the bug on Sepolia, you asked me why the tests didn't catch it. and It seems this was the reason.

Reply:
I think the operator part is mixed up but I do remember. This was caught in the audit, we were not expecting the final price to be higher and had to reverse that check. So this makes sense to me now, thanks.